spec/models/cheatsheet_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe Cheatsheet, type: :model do
+
+  before :context do
+    @user = User.find(3)
```

Comment:
Where's this user created at? You should look at maybe using factories for this kind of creation. `FactoryBot` (https://github.com/thoughtbot/factory_bot_rails) is the de facto standard.

Reply:
i created it in my rails console lol, i will look into the factory bot :D